pages/dkp/kommander/2.2/insights/insights-setup/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle: Insights setup and configuration
+title: Insights setup and configuration
+excerpt: Enable DKP Insights in managmenet and attached/managed clusters
+menuWeight: 20
+beta: false
+enterprise: false
+techPreview: true
+---
+
+DKP Insights is installed by default on your Management Cluster, and then you need to manually enable the DKP Insights Engine in order to begin viewing insight items for your environment.
+
+## Enable the DKP Insights engine
+
+To enable DKP Insights:
+
+1.  Apply yaml on your management cluster from the CLI:
+
+   ```bash
+   export WORKSPACE_NAMESPACE=kommander
+
+   kubectl apply -f - <<EOF
+   apiVersion: source.toolkit.fluxcd.io/v1beta1
+   kind: GitRepository
+   metadata:
+    name: insights-catalog-applications
+    namespace: ${WORKSPACE_NAMESPACE}
+    labels:
+     kommander.d2iq.io/gitapps-gitrepository-type: dkp
+     kommander.d2iq.io/workspace-default-catalog-repository: "true"
+   spec:
+    interval: 1m0s
+    ref:
+     branch: main
+    timeout: 20s
+    url: https://github.com/mesosphere/insights-catalog-applications
+   EOF
```

Comment:
Question for @kaiwalyajoshi - do we really need this variable substitution here, it seems like we only ever apply this config to the management cluster and thus the kommander namespace?

Also, how does one do this Airgapped?   If we don't have a solution for AG in 2.2, can I suggest we remove the mentions of airgapped for now?

I think the example would be simpler if we just 'hardcode' this for the kommander NS.

```suggestion
   kubectl apply -f - <<EOF
   apiVersion: source.toolkit.fluxcd.io/v1beta1
   kind: GitRepository
   metadata:
    name: insights-catalog-applications
    namespace: kommander
    labels:
     kommander.d2iq.io/gitapps-gitrepository-type: dkp
     kommander.d2iq.io/workspace-default-catalog-repository: "true"
   spec:
    interval: 1m0s
    ref:
     branch: main
    timeout: 20s
    url: https://github.com/mesosphere/insights-catalog-applications
   EOF
```